polymesh.properties

```diff
@@ -346,20 +347,36 @@ menu.preferences=Preferences
 menu.showSelectionOnPreview=Show Vertex Selection on Preview
 menu.liveUpdate=Live Update
 menu.boldEdges=Bold Edges
+menu.fitToAll=Fit to All
+menu.fitToSelection=Fit to Selection
+menu.showGrid=Grid
 mapping=Mapping:
 texture=Texture:
 scaleAll=Scale All Meshes
-exportImageFile=Export mapping to BMP file
-chooseExportImageFile=Choose BMP file
-width=Width:
-height=Height:
-file=File:
-choose=Choose
-chooseColor=Choose Color
-exportImageSize=Select the size of the image:
+
+#Export dialog and messages
+exportImageFile=Export mapping to PNG file
+chooseExportImageFile=Choose PNG file
+imageResolution=Image resolution:
+backgroundType=Background type:
+transparent=Transparent
+white=White
+textured=Textured
+textureOnly=Texture image only
+lineProperties=Line properties:
+softLines=Soft lines
+useMappingColor=Mapping color
+useBlack=Black
+exportImage=Export
+mappingSizeWarning=Some vertex coordinates are outside the texture image area.
+revert=Revert
+fileExtensionChanged=The file was saved as 
+
+# pieces
 pieceName=Mesh Piece Name
 enterPieceName=Enter new piece name:
 enterMappingName=Enter new mapping name:
+# 'mapping' is translated twice -- needs to be separated somehow
```

Comment:
The first one seems to have a colon at the end "Mapping:" Depending on where this is used, perhaps we need two different keys, for different purposes?

Reply:
So it is: One of them is for a the mapping menu, one is in a dialog.

Reply:
If the punctuation really makes a difference, could you do a small PR to differentiate them?

Reply:
Ok, so, 'mapping' is translated _three_ times: `menu.mapping`, `mapping` with colon and `mapping` as such. They are all there for a reason.

I found, where the last one is used and along with it some unfinished business in handling unfoldings.... I'll do a PR on the translations and see if there is an easy fix for the other problem (as an other PR if so).